### PR TITLE
Stack update `v2023-08-17` for `osx-xcode-14.3.x-edge`

### PR DESCRIPTION
**Pull request by bitrise-infrabot**

### CODE REVIEW

File: data/osx-xcode-14.3.x-edge/languages.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
           "name": "Ruby versions",
           "category": "languages",
           "version": "",
-          "raw_output": "3.0.0\n  3.0.3\n  3.0\n  3.1.3\n  3.1.4\n  3.1\n  3.2.2\n *3.2\n  3",
```
Comment (ofalvai) [inline data/osx-xcode-14.3.x-edge/languages.json:10]: https://bitrise.atlassian.net/browse/BE-1190

**Review by ofalvai — APPROVED** [review]

File: data/osx-xcode-14.3.x-edge/android-sdk.txt [hunk 2]
```diff
@@ -5,10 +5,12 @@
   build-tools;32.1.0-rc1                                                            | 32.1.0 rc1   | Android SDK Build-Tools 32.1-rc1   | build-tools/32.1.0-rc1                                                           
   cmake;3.10.2.4988404                                                              | 3.10.2       | CMake 3.10.2.4988404               | cmake/3.10.2.4988404                                                             
   cmake;3.22.1                                                                      | 3.22.1       | CMake 3.22.1                       | cmake/3.22.1                                                                     
-  emulator                                                                          | 31.3.14      | Android Emulator                   | emulator                                                                         
+  emulator                                                                          | 32.1.14      | Android Emulator                   | emulator
```
[the remark this answers is not in the document]
Comment (lpusok) [reply data/osx-xcode-14.3.x-edge/android-sdk.txt:8]: It looks like it is the newer version of the emulator, which is what we wanted, right?

[the remark this answers is not in the document]
Comment (lpusok) [reply data/osx-xcode-14.3.x-edge/android-sdk.txt:8]: Is it possibly a dependency of system images?

[the remark this answers is not in the document]
Comment (ofalvai) [reply data/osx-xcode-14.3.x-edge/android-sdk.txt:8]: We cleaned up system images too 😆 But good idea, maybe some other SDK package depends on this. My other guess would be Flutter, especially `flutter doctor` that we run during the build 🤔

**Pull request merged**